### ISSUE TITLE
TsProperty Name override, is itself overridden by config settings

### ISSUE THREAD

**Issue by PandaWood** [reinforced/Reinforced.Typings#57]
```
[JsonIgnore,TsProperty(Name = "ID", ForceNullable = true)]
protected virtual int _id { get { return 0; }}
```
Gives me below, when settings are for Camel Case, but I don't expect camel case settings to override an explicit setting of the "Name"
```
export interface IDomainObject
{	
	id?: number;
}
```

**Comment by pavel-b-novikov** [comment]
Crap. RT is even more logical than required :D

**Comment by pavel-b-novikov** [comment]
Fixed in version 1.4.2
[unit test](https://github.com/reinforced/Reinforced.Typings/blob/master/Reinforced.Typings.Tests/SpecificCases/SpecificTestCases.OverridenNamesNotCamelCased.cs)
